# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/UtilityProcessChild.cpp
# Commit: 70eab0533b00
# Full Hash: 70eab0533b00cee4ac31f7e3250d78d2a9a09758
# Author: Ray Kraesig <rkraesig@mozilla.com>
# Date: 2023-09-16 09:14:45
# Regressor Bug: 1837079
# File Overlap Count: 1
# Description:
#   Bug 1837079 - [7/10] Open Windows file picker out-of-process  r=gstoll,handyman,ipc-reviewers,nika
#   
#   When opening a new Windows file dialog, open it out-of-process if
#   possible. Fall back to opening it in-process if that fails. (This
#   behavior is configurable with a pref.)
# ==============================================================================

diff -r 488707a84723 -r 70eab0533b00 ipc/glue/UtilityProcessChild.cpp
--- a/ipc/glue/UtilityProcessChild.cpp	Fri Sep 15 18:10:01 2023 +0000
+++ b/ipc/glue/UtilityProcessChild.cpp	Fri Sep 15 18:10:01 2023 +0000
@@ -6,6 +6,7 @@
 #include "UtilityProcessChild.h"
 
 #include "mozilla/AppShutdown.h"
+#include "mozilla/Logging.h"
 #include "mozilla/dom/ContentParent.h"
 #include "mozilla/dom/JSOracleChild.h"
 #include "mozilla/dom/MemoryReportRequest.h"